rust/osm2lanes/src/tag/mod.rs

```diff
@@ -11,6 +11,8 @@ pub use key::TagKey;
 mod osm;
 pub use osm::{Highway, HighwayType, Lifecycle, HIGHWAY, LIFECYCLE};
 
+use crate::transform::{RoadWarnings, TagsToLanesMsg};
```

Comment:
it feels strange including these in this mod.

I would expect tag to eventually become its own crate or similar, with no dependencies on the osm2lanes transformation logic.

Reply:
Agreed, `get_parsed` is a convenience function for parsing a bunch of tags at the same time, and should probably live in `transform` somewhere. It was suggested that it be moved in here, but maybe it should be a function, not a method: `let l: Option<usize> = get_parsed_tag(tags, "lanes", &warnings)`; or a method on `RoadWarnings`: `let l: Option<usize> = warnings.parse_tag(tags, "lanes")`?